### PR TITLE
{chem}[intel/2015a] ABINIT 7.10.4 (REVIEW)

### PR DESCRIPTION
**Pull request by boegel**
(synced with #1551)

### CODE REVIEW

File: easybuild/easyconfigs/a/ABINIT/ABINIT-7.10.4-intel-2015a.eb [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+configopts += '--with-netcdf-incs="-I$EBROOTNETCDF/include -I$EBROOTNETCDFMINFORTRAN/include" '
+configopts += '--with-netcdf-libs="-L$EBROOTNETCDF/lib -lnetcdf -L$EBROOTNETCDFMINFORTRAN/lib -lnetcdff" '
+configopts += '--with-libxc-incs="-I$EBROOTLIBXC/include" --with-libxc-libs="-L$EBROOTLIBXC/lib -lxc" '
+
+dependencies = [
+    ('libxc', '2.2.2'),
+    ('netCDF', '4.3.2'),
+    ('netCDF-Fortran', '4.4.0'),
+]
+
```
Comment (wpoely86) [inline easybuild/easyconfigs/a/ABINIT/ABINIT-7.10.4-intel-2015a.eb]: sanity check?

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/c6bf1df3eb5122860814 for a full test report.

**Comment by boegel** [comment]
Jenkins: test this please

**Comment by boegel** [comment]
Going in, thanks for the review @wpoely86!

File: easybuild/easyconfigs/a/ABINIT/ABINIT-7.10.4-intel-2015a.eb (hunk 1 above)
Comment (boegel) [inline easybuild/easyconfigs/a/ABINIT/ABINIT-7.10.4-intel-2015a.eb]: added, thanks

**Pull request merged**